### PR TITLE
Fix performers section

### PR DESCRIPTION
**Pull request by ghost**
Changed the layout to 1-3-3. It fixes the problem for most of the devices. Excluding Iphone 4,4s,5,5s,6. The layout still seems broken on these device models with smaller screen sizes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,22 @@
 Breeze is Shiv Nadar University's annual techno-culturo fest.
 
 ## Link
+<<<<<<< HEAD
```
Comment (rhnvrm) [inline README.md:10]: Merge Conflicts

**Comment by ghost** [comment]
My bad...This had like around a couple of commits from my old ones...Idk why...

**Comment by ghost** [comment]
Will resolve it and then open a pR

**Comment by rhnvrm** [comment]
git rebase

**Comment by rhnvrm** [comment]
and force push

**Comment by ghost** [comment]
Force pushing will replace the new readme with my old copy of the readme !
@rhnvrm

**Comment by rhnvrm** [comment]
drop other commits, keep only 06a23d1

**Comment by ghost** [comment]
Trying to do it...But really, everything seems so fucked up right now ! I should probably delete the branch and redo it

**Comment by rhnvrm** [comment]
Dont worry, work on #23 

I'll apply your patch to gh-pages

**Comment by rhnvrm** [comment]
merged with ef4014 (https://github.com/snu-breeze/breeze-landing/commit/ef4014791676bd378e9fd6f088aff697da5055fd)

**Pull request closed without merging**